feriados_calculo.js

```diff
@@ -435,6 +435,24 @@ function obterFeriadosMunicipais(ano, uf, municipio) {
             diaDeCorpusChristi(),
             diaDaConscienciaNegra(),
         ];
+        case "SP/São Carlos": return [
+            { tipo: "MUNICIPAL", data: new Date(ano, JANEIRO, 2), descricao: "(ponto facultativo)" },
```

Comment:
Pelo que entendi, vários casos de "(ponto facultativo)" seriam emendas de feriados. Em São Carlos, a lei determina que todas as emendas de feriados são pontos facultativos? Isso vale apenas para funcionários públicos ou para todos os cidadãos?

Reply:
salve mestre! Então, peguei a relação da [lei municipal](http://www.saocarlos.sp.gov.br/index.php/conheca-sao-carlos/115443-feriados.html), onde aparece "ponto facultativo". Mas, pensando melhor, eu vou retirar esses pontos facultativos até o #2 for feito. ;-)